src/EditorFeatures/Core.Wpf/InlineRename/InlineRenameSession.cs

```diff
@@ -304,7 +303,11 @@ private void UpdateReferenceLocationsTask(JoinableTask<IInlineRenameLocationSet>
                 await ThreadingContext.JoinableTaskFactory.SwitchToMainThreadAsync(alwaysYield: true, _cancellationTokenSource.Token);
                 _cancellationTokenSource.Token.ThrowIfCancellationRequested();
 
-                RaiseSessionSpansUpdated(inlineRenameLocations.Locations.ToImmutableArray());
+                var locations = _workspace.IgnoreUnchangeableDocumentsWhenApplyingChanges
```

Comment:
@jasonmalinowski This change only affect the text displayed in UI, it doesn't actually filter out unchangeable documents from all references. I will add a comment to make is clear and leave it here as discussed offline.